Makefile

```diff
@@ -140,7 +140,14 @@ $(KCP_APIGEN_GEN):
 	pushd . && cd sdk && GOBIN=$(TOOLS_GOBIN_DIR) go install ./cmd/apigen && popd
 
 lint: $(GOLANGCI_LINT) $(LOGCHECK) ## Verify lint
-	$(GOLANGCI_LINT) run --timeout 20m ./...
+	echo "Linting root module..."; \
```

Comment:
This gonna hurt CI, maybe just keep root module and cli

Reply:
but we never ran lint against all the modules, this PR fixes all the issues across the whole codebase